src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const threeDayDiscount = 20;
+  const sevenDayDiscount = 50;
+  const basePrice = 40;
+
+  if (days >= 7) {
```

Comment:
The condition `days >= 7` uses a hardcoded literal. Replace it with a constant that explains this value. For example, `const LONG_TERM = 7;` as suggested in the checklist item 2.